### DISCUSSION TITLE
Getting None content_type for object using the Python API

### DISCUSSION THREAD

**Discussion by sprior** [minio/minio#17503] [Q&A] [answered]
I can't seem to retrieve the content-type for an object using the Python API.  I just created a new up to date minio instance in Docker and used the sample python example to create an object:
```
client.fput_object(
        "asiatrip", "DSC_4829.jpg", "DSC_4829.jpg", content_type='image/jpeg'
    )
```
    
Then I'm listing those objects with:

```
objects = client.list_objects('asiatrip', recursive=True)
    for obj in objects:
        print(obj.bucket_name, obj.object_name.encode('utf-8'), obj.last_modified,
            obj.etag, obj.size, obj.content_type)
```
            
But the obj.content_type is None.Using the minio gui on localhost:9001 and looking at the same object I see in the metadata section that it has the correct content_type so the issue appears to be in the query not the put.Any ideas what I'm missing?  Is there some permission the account needs?

**Comment by harshavardhana** [comment, marked as answer]
content type is never returned in the listObjects API in S3

This is normal. https://docs.aws.amazon.com/AmazonS3/latest/API/API_Object.html

**Reply by sprior** [reply]
Ok then, but then I'd say the example code here at line 31 (https://github.com/minio/minio-py/blob/release/examples/list_objects.py) which tries to print the content type from listed objects is wrong and should be fixed.

**Reply by harshavardhana** [reply]
Feel free to send a fix @sprior we will happily accept it.
